lib/types/types.dart

```diff
@@ -56,9 +56,14 @@ class TextNode extends Node<String> {
   TextNode(dynamic node) {
     value = node['value'] ?? '';
     _nodeType = node['nodeType'] ?? '';
-    node['marks']?.map((mark) {
-      marks.add(Mark(mark['type']));
-    });
+    List<dynamic> markList = node['marks'] ?? [];
+    // Adding types to marks
+    if (markList.isNotEmpty) {
+      var typeList = markList.map((e) => e['type']).toList();
+      for (var i = 0; i < typeList.length; i++) {
+        marks.add(Mark(typeList[i]));
+      }
+    }
```

Comment:
this was resolved in a different way, see:
https://github.com/Kumanu/contentful-rich-text-flutter/pull/13/files#diff-6ffa5009051dcdc55483857b2161e608ede882e046899aa1778dc3757a76d5f6